src/main/java/io/jenkins/plugins/appcenter/task/internal/DistributeResourceTask.java

```diff
@@ -99,8 +101,15 @@ private String parseReleaseNotesFromFile(@Nonnull String pathToReleaseNotes) {
 
     }
 
+    private static BuildInfo CreateBuildInfo(@Nonnull UploadRequest request) {
```

Comment:
We should also add the appropriate nullability information here. i.e. `@Nullable`